AccessHandle.md

```diff
@@ -215,10 +215,18 @@ convenient, both from a specification and performance point of view.
 
 ## Trying It Out
 
-A prototype of access handles is available in Chrome Canary (from at least
-version `94.0.4597.0`). To enable it, launch Chrome with the
+A prototype of access handles is available in Chrome. If you're using version
```

Comment:
Maybe clarify that only sync access handles are available? Also, the term "sync access handles" never really appears in the remainder of the doc, it's just the name of the method.

Reply:
Added a note and clarified what sync access handles are.